neo/neo.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Copyright>2015-2019 The Neo Project</Copyright>
     <AssemblyTitle>Neo</AssemblyTitle>
-    <Version>2.10.3</Version>
+    <Version>2.10.3-neox-preview1</Version>
```

Comment:
if merged perhaps it will be something like 2.11.0, yes?

Reply:
Wil do.

Reply:
#1810

Reply:
Merged